lexer.js

```diff
@@ -1439,4 +1439,5 @@ function isExpressionTerminator (curPos) {
 const initPromise = Promise.resolve();
 
 module.exports.init = () => initPromise;
+module.exports.initSync = () => {};
```

Comment:
I am not sure why `dist/lexer.js` was never documented in the README, despite being what Node.js core has been using. But anyway this is added to lexer.js for parity.